app/schemas/schema.js

```diff
@@ -1,112 +1,230 @@
 const schema = {
-  "title": "Request to Participate",
   "type": "object",
   "required": [
-    "organization",
-    "streetNumber",
-    "streetName",
-    "city",
-    "postalCode"
+    // "Organization",
+    // "Registration Number",
+    // "Street Number",
+    // "Street Name",
+    // "City",
+    // "Postal Code",
+    // "Primary Contact",
+    // "Postion",
+    // "Contact Email",
+    // "Telephone",
+    // "Zone Feedback",
+    // "Wired Broadband",
+    // "Fixed Wireless",
+    // "Last Mile Support",
+    // "Anticipated Projects",
+    // "Underserved Areas",
+    // "Satellite Proposal",
+    // "Capital Cost Funding",
+    // "Capital Cost Explanation",
+    // "Last Mile Broadband",
+    // "Last Mile Fixed Wireless",
+    // "Last Mile New Backbone",
+    // "Highway Cellular"
   ],
   "properties": {
-    "Applicant Profile:": {
+    "Organization Profile": {
+      "title": "Organization Profile",
+      "type": "object",
       "properties": {
-        "organization": {
-          "title": "Applicant organization (legal name):",
+        "Organization": {
+          "title": "Organization name (legal name)",
           "type": "string"
         },
-        "registrationNumber": {
+        "Registration Number": {
           "type": "string",
-          "title": "Band number, society number, or business registration number (if applicable):"
+          "title": "Band number, society number, or business registration number (optional)"
         },
-        "streetNumber": {
+        "Unit Number": {
           "type": "number",
-          "title": "Street number:"
+          "title": "Unit number (optional)"
         },
-        "streetName": {
+        "Street Number": {
+          "type": "number",
+          "title": "Street number"
+        },
+        "PO Box": {
+          "type": "number",
+          "title": "PO Box (optional)"
+        },
+        "Street Name": {
           "type": "string",
-          "title": "Street name:"
+          "title": "Street name"
         },
-        "city": {
+        "City": {
           "type": "string",
-          "title": "City:"
+          "title": "City"
         },
-        "postalCode": {
+        "Postal Code": {
           "type": "string",
-          "title": "Postal code:"
+          "title": "Postal code (H0H 0H0)"
         }
       },
-      "type": "object"
     },
-    "Primary Contact": {
+    "Contact Information": {
+      "title": "Contact Information",
```

Comment:
Might we want a description of whose contact information we're collecting?

Reply:
How so? I was implementing from the design.